packages/workflow/src/interceptors.ts

```diff
@@ -18,6 +18,14 @@ export interface WorkflowExecuteInput {
   readonly headers: Headers;
 }
 
+/** Input for WorkflowInboundCallsInterceptor.handleUpdate and
+ * WorkflowInboundCallsInterceptor.validateUpdate */
+export interface UpdateInput {
+  readonly name: string;
+  readonly args: unknown[];
```

Comment:
Which is preferred here, `unknown[]` or `any[]`?

Reply:
`unknown` is generally prefered.

Essentially:
- `unknown` means you don't know, but want to remain safe, so you can't do anything with those;
- `any` means you don't know and you don't care, so the compiler will let you do anything with it (ie. unsafe).